pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -232,3 +245,31 @@ social:
       caption.fadeOut();
     });
   });
+
+  function startLATAMVideo()
+  {
+    if (videoStarted)
+      return;
+
+    $("<iframe allowfullscreen frameborder='0' src='https://www.youtube-nocookie.com/embed/OpgWbXthMlQ?iv_load_policy=3&amp;rel=0&amp;autohide=1&amp;showinfo=0&amp;autoplay=1&cc_load_policy=1' style='position:absolute; top: 0; left: 0; width: 100%; height: 100%; background-color:black;'></iframe>").appendTo("#videodiv");
```

Comment:
I'm still waiting on the video code - right now this plays a Finding Nemo clip 🐠. Double check that I update when the video is ready!